### DISCUSSION TITLE
How to install on Insider version?

### DISCUSSION THREAD

**Discussion by AX-Procyon** [valinet/ExplorerPatcher#282] [Q&A] [answered]
I am on Insider (Dev Channel). I followed the instructions and placed the DLL into C:\Windows and restarted Explorer. However, it says that the files required are not available to my version of Windows. Nothing was downloaded (that I'm aware of) and the whole app just doesn't function. How do I bypass the limit or manually download the required files? I know it will be uncharted waters but I'm willing to take the risk. Thanks in advance!

**Comment by Andre-10** [comment, marked as answer]
Have it installed on Insider Dev build (latest), and no problems here. Maybe it's language dependent?

Btw, working great.. no problems here. Big Thanks to developer
